build.gradle.kts

```diff
@@ -27,18 +23,14 @@ allprojects {
             targetCompatibility = JavaVersion.VERSION_11.toString()
         }
 
+        val username by variable("username")
```

Comment:
Shouldn't this also be `goplay.username`?

Reply:
What is in `local.properties`? I thought enventually we'd have:

```kotlin
val username by variable("username")

listOf("goplay", "vtm").forEach { brand ->
  enviroment("${brand}.username, username)
}
```

Or should it rather be:

```kotlin
listOf("goplay", "vtm").forEach { brand ->
  val username by variable("${brand}.username")
  enviroment("${brand}.username, username)
}
```

Reply:
If the later, then yes we should change it to `goplay.username`

Reply:
We had following structure. 

```
vrtnu.username=
vrtnu.password=

vtmgo.username=
vtmgo.password=

goplay.username=
goplay.password=
```

Unless we have 1 dummy account where all username and passwords are equal for all services.

Reply:
Alright, makes sense. Let me update the code.